src/monitor/cpu_set_monitor.cpp

```diff
@@ -58,7 +58,7 @@ void CpuSetMonitor::run()
     {
         monitor_elem.second.start();
     }
-    trace_.register_tids(read_all_tid_exe());
+    trace_.add_threads(get_comms_for_running_processes());
```

Comment:
Why are we doing this twice?

https://github.com/tud-zih-energy/lo2s/pull/84/files#diff-6b6f0e5f9bcb197c02e9fafecaaff995R88